modules/openideal_idea/src/Plugin/Block/OpenidealIdeaUpdateInfo.php

```diff
@@ -110,10 +121,15 @@ public function build() {
         'access' => $this->configuration['use_updated'],
       ];
 
-      $member = $this->helper->getGroupMember($this->currentUser, $node);
-      if ($member && $member->hasPermission('update any group_node:idea entity')) {
-        $link = Link::createFromRoute($this->t('Edit'), 'entity.node.edit_form', ['node' => $node->id()])->toString();
-        $build['#content']['edit'] = $link;
+      $group = $this->helper->getGroupFromNode($node);
+      if (($group && $this->groupPermissionChecker->hasPermissionInGroup('update any group_node:idea content', $this->currentUser, $group)) ||
+        (!$group && $node->access('update', $this->currentUser))) {
```

Comment:
Do we have any cases when idea is outside of the group?

Reply:
Heh, we need probably to rename this block, because it's being used for almost every content type, and need to check node access if it's non-groupable node.

Reply:
Let's check if we can simplify this condition. Probably one permission/access check will be enough to check both cases (group content vs individual content).